tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -5825,7 +5812,7 @@ TEST_F( EnergyPlusFixture, TubularDaylightDiffuserCount )
 	ASSERT_FALSE( process_idf( idf_objects ) );
 
 	OutputProcessor::TimeValue.allocate( 2 );
-	DataGlobals::DDOnlySimulation = true;
+	//DataGlobals::DDOnlySimulation = true;
```

Comment:
Very minor quibble about some commented code being left around here.  Only worry about cleaning it if there is other work needed to be committed.